src/api/js/src/high-level/high-level.test.ts

```diff
@@ -0,0 +1,383 @@
+import assert from 'assert';
```

Comment:
FWIW: We are in the process of creating a supported version of the z3 guide that uses the SMTLIB front-end. It could also mix in js code, not previously possible.

Reply:
Interesting, do you mean a guide with interactive examples?

Reply:
right.
Several years ago there were two guides exposed: one using SMTLIB2 formulas written into a text box. The other wrapped around Python. They used different backends and required hosting. This had to be discontinued.
With wasm bindings there need not be two different tutorials. The self-contained web pages can host examples in text and examples in code.